tests/helper/helper_podman.go

```diff
@@ -7,13 +7,30 @@ import (
 	"os/exec"
 	"path/filepath"
 	"regexp"
+	"strconv"
 	"strings"
 
 	. "github.com/onsi/gomega"
 	"github.com/redhat-developer/odo/pkg/podman"
 )
 
+func getBooleanValueFromEnvVar(envvar string, defaultValue bool) bool {
+	strVal := os.Getenv("PODMAN_USE_NAMESPACES")
+	if strVal == "" {
+		return defaultValue
+	}
+	boolValue, err := strconv.ParseBool(strVal)
```

Comment:
```suggestion
	boolValue, err := strconv.ParseBool(strVal)
```

If the string is empty, it will return the same error as if the string were invalid.
```sh
false strconv.ParseBool: parsing "": invalid syntax
false strconv.ParseBool: parsing "somthing": invalid syntax
```